src/main/java/com/openshift/jenkins/plugins/pipeline/OpenShiftScalerPostAction.java

```diff
@@ -55,111 +57,173 @@ public String getVerifyReplicaCount() {
-         *
-         * @param value
-         *      This parameter receives the value that the user has typed.
-         * @return
-         *      Indicates the outcome of the validation. This is sent to the browser.
-         *      <p>
-         *      Note that returning {@link FormValidation#error(String)} does not
-         *      prevent the form from being saved. It just means that a message
-         *      will be displayed to the user. 
-         */
-        public FormValidation doCheckApiURL(@QueryParameter String value)
-                throws IOException, ServletException {
-        	return ParamVerify.doCheckApiURL(value);
-        }
-
-        public FormValidation doCheckDepCfg(@QueryParameter String value)
-                throws IOException, ServletException {
-        	return ParamVerify.doCheckDepCfg(value);
-        }
-
-        public FormValidation doCheckNamespace(@QueryParameter String value)
-                throws IOException, ServletException {
-        	return ParamVerify.doCheckNamespace(value);
-        }
-        
-        
-        public FormValidation doCheckReplicaCount(@QueryParameter String value)
-                throws IOException, ServletException {
-        	return ParamVerify.doCheckReplicaCountRequired(value);
-        }
-
-        public FormValidation doCheckAuthToken(@QueryParameter String value)
-                throws IOException, ServletException {
-        	return ParamVerify.doCheckToken(value);
-        }
-
-        public boolean isApplicable(Class<? extends AbstractProject> aClass) {
-            // Indicates that this builder can be used with all kinds of project types 
-            return true;
-        }
-
-        /**
-         * This human readable name is used in the configuration screen.
-         */
-        public String getDisplayName() {
-            return DISPLAY_NAME;
-        }
-        
-        public long getWait() {
-        	return wait;
-        }
-
-        @Override
-        public boolean configure(StaplerRequest req, JSONObject formData) throws FormException {
-            // To persist global configuration information,
-            // pull info from formData, set appropriate instance field (which should have a getter), and call save().
-        	wait = formData.getLong("wait");
-            save();
-            return super.configure(req,formData);
-        }
-
-    }
 
+	// Overridden for better type safety.
+	// If your plugin doesn't really define any property on Descriptor,
+	// you don't have to do this.
+	@Override
+	public DescriptorImpl getDescriptor() {
+		return (DescriptorImpl)super.getDescriptor();
+	}
+
+	/**
+	 * Descriptor for {@link OpenShiftScalerPostAction}. Used as a singleton.
+	 * The class is marked as public so that it can be accessed from views.
+	 *
+	 */
+	@Extension // This indicates to Jenkins that this is an implementation of an extension point.
+	public static final class DescriptorImpl extends BuildStepDescriptor<Publisher> {
```

Comment:
I had forgotten about editing this class.  Intellij is telling me that the class is never used, is that correct?